poetry.lock

```diff
@@ -682,7 +682,7 @@ doc = ["Sphinx", "sphinx-rtd-theme"]
 name = "setuptools"
 version = "65.6.3"
```

Comment:
Considering poetry lock already has an entry for a version that matches the constraint in pyproject, do we absolutely need to explicitly add setuptools to pyproject?

Reply:
Nm, I saw the change in category from dev to main too late